ConfigData/phase GEMS Comparison.py

```diff
@@ -261,6 +356,161 @@ def GEMSChiPlot(AtPct, Protosolar):
 
     return Chondricity, ElementSigma, GEMSChiSqRed, DOF
 
+def GEMSChiPlotWeb(AtPct, Protosolar):
```

Comment:
There should be only one ChiPlot function which is the same for web and wxpython.  In both cases, we should return the plotly figure.  If in wxpython, the fig.show auto-loads a browser, I think.  If in web, then it can be embedded in the gui.  But you don't want to fork code like this or a single change requires changes in two locations.